index.html

```diff
@@ -1,47 +1,160 @@
+            their robot in Python to compete in the competition.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/glue-gun.jpg' | prepend: site.baseurl }}" alt="" />
+          <h3>Building a robot</h3>
+          <p>
+            There's nothing quite like building something and bringing it to
+            life. Using our kit, teams will build a fully autonomous robot. The
+            best teams pour their imagination into the design and the look of
+            their robots.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row vertically-centered-row">
+        <div class="column m-9-12">
+          <blockquote class="text-italic">
+            Student Robotics provided me with the invaluable experience of working on a engineering project with a team.
+            I learned not only good engineering principals but the importance of being able to clearly communicate my
+            thoughts and reasoning.
+            Competing in Student Robotics opened the door to me directly entering the world of work as a Software
+            Engineer after completing sixth form.
+          </blockquote>
+          <p class="text-heavy text-center">— James Seden Smith</p>
+          <h5 class="text-center">
+            Student Robotics 2015 and 2016<br>
+            Team CLY
+          </h5>
+        </div>
+        <div class="column m-3-12 horizontally-centered-column">
+          <img class="testomonial-avatar"
+            src="{{ '/images/advertising/headshots/james-seden-smith.jpg' | prepend: site.baseurl }}" />
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel panel-highlight panel-thin">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center top-spacing">Competitor feedback from the last two years</h2>
+      <div class="row">
+        <div class="column m-4-12">
+          <p class="statistic">100%</p>
+          <p class="statistic-label">would recommend Student Robotics</p>
+        </div>
+        <div class="column m-4-12">
+          <p class="statistic">88%</p>
+          <p class="statistic-label">are now highly interested in a STEM career</p>
+        </div>
+        <div class="column m-4-12">
+          <p class="statistic">97%</p>
+          <p class="statistic-label">plan to study a STEM degree at university</p>
+        </div>
       </div>
     </div>
   </div>
-  <div class="fixed-width">
-    <div class="row">
-      <div class="sidebar column m-4-12">
-        <div class="card">
-          <h3 class="card-heading">Upcoming Events</h3>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width text-center">
+      <div class="row">
+        <div class="column m-8-12">
```

Comment:
Question: Is there a reason posts is twice as wide as events? 50:50 might look nicer? Or perhaps use this weighted layout just when there are no events?